custom/covid/management/commands/update_all_activity_complete_date.py

```diff
@@ -56,7 +56,7 @@ def handle(self, domain, **options):
                 domain=domain,
                 update_fn=_correct_bad_property,
                 case_ids=with_progress_bar(bad_case_ids, oneline=False),
-                form_meta=SystemFormMeta.for_script(__name__, options['username']),
+                form_meta=SystemFormMeta.for_script(__name__, options.get('username')),
```

Comment:
I believe `for_script` will throw an error if username is not provided.  Also FWIW with command options all keys will be present in the dict, even when not specified so `options['username']` is fine, it'll just return `None`

Reply:
oops, yes it does. 595f36c912677231f61190317e7e7757cb98e8d5 fixes this up.